recipes/cofi-espresso/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cofi-espresso" %}
+{% set version = "0.0.2.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cofi_espresso-{{ version }}.tar.gz
+  sha256: c874b36076ea646a3f9d19e8896198a28c8a1db28ad704c7d120c044963d1fa6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - $RPATH/libgfortran.so.5
+    - /lib64/libm.so.6
+    - /lib64/libc.so.6
+    - lib/libgcc_s.so.1
+    - lib/libquadmath.so.0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - versioningit
+  run:
+    - python >=3.6
+    - numpy >=1.18
+    - scipy >=1.0.0
+    - matplotlib-base >=3.1
+    - tqdm >=4.0
+
+test:
+  imports:
+    - cofi_espresso
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/inlab-geo/espresso
+  summary: Earth Science PRoblems for the Evaluation of Strategies, Solvers and Optimizers
+  license: 'BSD-2-Clause'
+  license_file: LICENCE
```

Comment:
This should also include the list of vendored dependencies under https://github.com/inlab-geo/espresso/tree/main/contrib.

Reply:
Thanks for the review and comment! Did you mean the license files under all the `contrib`'s sub-folders? Would you think a regular expression `contrib/*/LICENCE` will work? If not, when I add a sub-folder into `contrib` in the future, is it the easiest way to raise a pull request in the feedstock to update this field?

Reply:
Would it work if I add the GitHub links with titles to all the other license files from this main license file?

Reply:
Not sure about the glob there, but PRs in feedstock would work for these changes.